src/html_support_files/dune

```diff
@@ -5,34 +5,32 @@
  (aliases runtest support-files)
  (enabled_if
   (> %{ocaml_version} 4.08))
+ (target odoc_html_support_files.ml)
```

Comment:
odoc.opam is not uptodate as `crunch` is still listed `with-test`. It would be nice to test various versions of crunch and find an eventual lower bound.